lib/express.js

```diff
@@ -25,17 +29,15 @@ function config (app, { showLoginPage, serviceProvider, idpConfig }) {
       const relayState = encodeURIComponent(req.query.Target)
       if (showLoginPage) {
         const identities = assertions.identities[idp]
-        const body = identities
+        const someValues = identities
           .map((value, index) => {
             const samlArt = samlArtifact(idpConfig[idp].id, index)
             const assertURL =
               `${idpConfig[idp].assertEndpoint}?SAMLart=${samlArt}&RelayState=${relayState}`
-            return `
-              <p>Click to login as <a href="${assertURL}">${value}</a></p>
-            `
+            assets.identity[index] = {'id': `${value}`, 'URL': `${assertURL}`}
           })
-          .join('\n')
-        res.send(`<html><body>${body}</body></html>`)
+        const response = render(LOGIN_TEMPLATE, assets.identity)
```

Comment:
As noted, the second argument should be changed to... something else. It does seem that `someValues` can be used as a viable alternative, with the appropriate mapping function...